vectorsearch/params/radial_search/add_radial_threshold_to_dataset.py

```diff
@@ -0,0 +1,89 @@
+import h5py
```

Comment:
should this be moved to new folder called _tools? @IanHoang @gkamat i see geoshape contains [_tools](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/geoshape) to store scripts. is this right place to move helper methods?

Reply:
@VijayanB yes, we can move this script to a separate folder called `_tools` to remain consistent with workloads like nyc_taxis and geoshape

Reply:
Moved it to `_tools` folder